client-src/elements/chromedash-feature-page.ts

```diff
@@ -119,6 +127,56 @@ export class ChromedashFeaturePage extends LitElement {
     return this.feature && Object.keys(this.feature).length !== 0;
   }
 
+  isFeatureUpcoming(channels, stages: Array<StageDict>) {
```

Comment:
I think this would be more clear if the method was named something like `findClosestShippingDate` or `calcUpcoming`.  When a method is named `is*`, I expect it to be a function that return a boolean and has no side-effects.

Reply:
Good point! Done